api_emulator/redfish/ResourceZone_api.py

```diff
@@ -87,7 +87,7 @@ def __init__(self):
         self.config = {
             '@odata.context': self.rb + '$metadata#ResourceZoneCollection.ResourceZoneCollection',
             '@odata.id': self.rb + 'CompositionService/ResourceZones',
-            '@odata.type': '#ResourceZoneCollection.1.0.0.ResourceZoneCollection',
+            '@odata.type': '#ResourceZoneCollection.ResourceZoneCollection',
```

Comment:
This should just be `'#ZoneCollection.ZoneCollection'`; there is no "ResourceZoneCollection" resource.

Reply:
OK - I didn't fix any dive in and fix several of these that were using outdated schema references (e.g., pcie_switches).  I am making those changes over in the Swordfish_API_Emulator, adding Storage correctly, moving drives under Chassis (since drives and Volumes are already instrumented there). For this one, I was just fixing the incorrect formatting.  But I can change the reference completely